MavenArtifacts.md

```diff
@@ -69,12 +69,30 @@ All artifacts are based at `edu.wpi.first.artifactname` in the repository.
 
 * wpiutil
 
+* wpigui
+  *imgui
```

Comment:
A space is required after the asterisk for this to be correctly interpretted as a bulleted list.